coral-component-shell/src/scripts/ShellHelp.js

```diff
@@ -64,6 +64,7 @@ class ShellHelp extends BaseComponent(HTMLElement) {
         itemBaseTagName: 'a',
         container: this._elements.items
       });
+      this._elements.items.setAttribute("role", "list");
```

Comment:
I think role can be added in template https://github.com/adobe/coral-spectrum/blob/master/coral-component-shell/src/templates/help.html#L7 rather than in js

Reply:
Moved role list to the `help.html` file onto this new PR (https://jira.corp.adobe.com/browse/SITES-10929). Can you please close this by the reason that Stefan is no longer working on this project and no one from the team can interact with this?
@Pareesh